components/automate-chef-io/content/docs/asset-store.md

```diff
@@ -1,33 +1,37 @@
 +++
-title = "Asset Store"
+title = "Profiles"
 description = "Select and Install Compliance Profiles"
 date = 2018-03-26T16:02:53-07:00
 draft = false
 bref = ""
 toc = true
+[menu]
+  [menu.docs]
+    parent = "compliance"
+    weight = 30
 +++
 
-## About the Asset Store
+## Overview
 
-Find Chef Automate compliance profiles under the **Asset Store** tab.
+Navigate to compliance profiles by using the **Compliance** tab and then selecting the Profiles page.
 Compliance profiles help you secure your infrastructure continuously.
 Chef Automate compliance profiles translate CIS Benchmarks and other security standards in easily readable policy.
 Chef Automate uses the InSpec language to turn compliance into code.
-You can install and download one of our 100+ ready-to-use compliance profiles from the Asset Store--or upload your own custom profiles.
+You can install and download one of our 100+ ready-to-use compliance profiles from the Profiles--or upload your own custom profiles.
 
-![Chef Automate Asset Store](/images/docs/asset-store-installed.png)
+![Chef Automate Profiles](/images/docs/asset-store-installed.png)
 
-## Using the Asset Store
+## Using the Profiles
 
-Navigate to the profiles page by selecting the **Asset Store** tab at the top of the Chef Automate screen or by heading to `https://{{< example_fqdn "automate" >}}/profiles`.
+Navigate to the profiles page by selecting the **Profiles** tab at the top of the Chef Automate screen or by heading to `https://{{< example_fqdn "automate" >}}/profiles`.
 
-The Asset Store has a search bar and two views, the _Profiles_ page, which displays the profiles you've installed
+The Profiles has a search bar and two views, the _Profiles_ page, which displays the profiles you've installed
```

Comment:
I didn't forsee "the" being a consistent issue when I suggested replacing _Asset Store_ with _Profiles_...